plugins/src/core/ddl.ts

```diff
@@ -0,0 +1,23 @@
+export interface DeepLinkContent {
+  title?: string;
+  description?: string;
+}
+export interface DeepLinkData {
+  data: { [key: string]: any };
```

Comment:
Can possibly use `Record<string,any>` for these types of declarations?

Reply:
updated. what's the benefit, only slightly neater syntax?

Reply:
Yeah, just a nice utility type and a bit easier to type out (I think).

https://www.typescriptlang.org/docs/handbook/utility-types.html